x-pack/plugins/alerting/public/pages/maintenance_windows/components/schema.tsx

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
This file can just be a typescript file (`.ts`) since there's no JSX in it

Reply:
Resolved in this commit e408455440054bd94d83fc8142a0b01a77860a5c